bokeh/models/axes.py

```diff
@@ -123,13 +123,8 @@ class Axis(GuideRenderer):
     of ticks.
     """)
 
-    axis_label = Nullable(String, default="", help="""
-    A text label for the axis, displayed parallel to the axis rule.
-
-    .. note::
-        LaTeX notation is not currently supported; please see
-        :bokeh-issue:`647` to track progress or contribute.
-
+    axis_label = Either(Null, String, Instance(MathText), default="", help="""
```

Comment:
@mattpap what would you think about using `Nullable(Either(String, Instance(MathText), ...)` in cases like this? It seems like consistently keeping `Nullable` at the outermost level would be nice, and potentially a slight benefit for documentation at some point.

Reply:
@IuryPiva can you try out the alternate property definition with `Nullable`? If that works (I think it will) then I would say that is preferable throughout. That's my only remaining comment.

Reply:
Done